src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -413,7 +413,7 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
     val draftAdjudicationResponse = intTestData.startNewAdjudication(testAdjudication)
 
     webTestClient.get()
-      .uri("/draft-adjudications/my/agency/${IntegrationTestData.ADJUDICATION_1.agencyId}/?startDate=2020-12-01")
```

Comment:
Removed `/?` as Spring stopped liking it - see [test failure](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-manage-adjudications-api/2036/workflows/4ee3c333-b79a-47f9-a9a3-22be98384262/jobs/5051)